examples/deployment/kubernetes/create.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#
+# This script (optionally) creates and then prepares a Google Cloud project to host a
+# Trillian instance using Kubernetes.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source ${DIR}/config.sh
+
```

Comment:
Worth adding a check on `gcloud` being in `PATH` at the top?

Reply:
done